WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -0,0 +1,75 @@
+<table class="table table-striped table-condensed admin-competitions">
+  <% show_tbody_message = title.include?("Recent") ? "hide" : "show" %>
```

Comment:
`show_tbody_message = "hide"` looks kinda weird, doesn't it? How do you like `toggle_tbody_message`?